modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/actions/PreIssueAccessTokenClientCredentialsGrantTestCase.java

```diff
@@ -179,34 +212,57 @@ protected boolean isRedirectable(String method) {
         OpenIDConnectConfiguration oidcConfig = getOIDCInboundDetailsOfApplication(applicationId);
         clientId = oidcConfig.getClientId();
         clientSecret = oidcConfig.getClientSecret();
+        subjectType = oidcConfig.getSubject().getSubjectType();
+        tokenType = oidcConfig.getAccessToken().getType();
+
         if (!CarbonUtils.isLegacyAuthzRuntimeEnabled()) {
-            authorizeSystemAPIs(applicationId, new ArrayList<>(Arrays.asList(SCIM2_USERS_API, ACTIONS_API,
-                    APPLICATION_MANAGEMENT_API, API_RESOURCE_MANAGEMENT_API)));
+            authorizeSystemAPIs(applicationId, Collections.singletonList(SCIM2_USERS_API));
         }
         domainAPIId = createDomainAPI(EXTERNAL_SERVICE_NAME, EXTERNAL_SERVICE_URI, customScopes);
         authorizeDomainAPIs(applicationId, domainAPIId, customScopes);
-
         addUserWithRole(applicationId, customScopes);
 
-        actionsMockServer = new ActionsMockServer();
+        requestedScopes = new ArrayList<>();
+        Collections.addAll(requestedScopes,
+                INTERNAL_ORG_USER_MANAGEMENT_LIST,
+                INTERNAL_ORG_USER_MANAGEMENT_VIEW,
+                INTERNAL_ORG_USER_MANAGEMENT_CREATE,
+                INTERNAL_ORG_USER_MANAGEMENT_UPDATE,
+                INTERNAL_ORG_USER_MANAGEMENT_DELETE);
+        requestedScopes.addAll(customScopes);
 
-        actionsMockServer.createMockServer(MOCK_SERVER_ENDPOINT);
         actionId = createPreIssueAccessTokenAction();
+
+        try {
+            Thread.sleep(3600);
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+        }
```

Comment:
why do we have this ?

Reply:
This was removed with e641385fd8a50b37b7b3090ff41135d479266800